### PR TITLE
cards for low pt bin of pt-binned V+jets samples

### PR DESCRIPTION
**Pull request by idebruyn**
50-100 GeV boson pt bin for
- DYJetsToLL
- DYJetsToNuNu
- GJets
- WJetsToLNu

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/GJets_Gpt_1j_5f_NLO/GJets_1j_Gpt-50To100_5f_NLO/GJets_1j_Gpt-50To100_5f_NLO_run_card.dat [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    1   = lpp1    ! beam 1 type (0 = no PDF)
+    1   = lpp2    ! beam 2 type (0 = no PDF)
+ 6500   = ebeam1  ! beam 1 energy in GeV
+ 6500   = ebeam2  ! beam 2 energy in GeV
+#***********************************************************************
+# PDF choice: this automatically fixes also alpha_s(MZ) and its evol.  *
+#***********************************************************************
+ lhapdf    = pdlabel   ! PDF set                       
+# 292200    = lhaid     ! if pdlabel=lhapdf, this is the lhapdf number              
+ 260000    = lhaid     ! if pdlabel=lhapdf, this is the lhapdf number
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/GJets_Gpt_1j_5f_NLO/GJets_1j_Gpt-50To100_5f_NLO/GJets_1j_Gpt-50To100_5f_NLO_run_card.dat]: this pdf looks like a 5F

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/GJets_Gpt_1j_5f_NLO/GJets_1j_Gpt-50To100_5f_NLO/GJets_1j_Gpt-50To100_5f_NLO_run_card.dat [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+ 0.4  = R0gamma   ! Radius of isolation code
+ 1.0  = xn        ! n parameter of eq.(3.4) in hep-ph/9801442
+ 1.0  = epsgamma  ! epsilon_gamma parameter of eq.(3.4) in hep-ph/9801442
+ .true.  = isoEM  ! isolate photons from EM energy (photons and leptons)
+#***********************************************************************
+# Maximal PDG code for quark to be considered a jet when applying cuts.*
+# At least all massless quarks of the model should be included here.   *
+#***********************************************************************
+# 5 = maxjetflavor
+ 4 = maxjetflavor
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/GJets_Gpt_1j_5f_NLO/GJets_1j_Gpt-50To100_5f_NLO/GJets_1j_Gpt-50To100_5f_NLO_run_card.dat]: but this is set to 4F.
Am I missing anything?

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/GJets_Gpt_1j_5f_NLO/GJets_1j_Gpt-50To100_5f_NLO/GJets_1j_Gpt-50To100_5f_NLO_run_card.dat (hunk 1 above)
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/GJets_Gpt_1j_5f_NLO/GJets_1j_Gpt-50To100_5f_NLO/GJets_1j_Gpt-50To100_5f_NLO_run_card.dat]: (btw, why different wrt the other cards? consistency with the other vpt binned samples?)

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/GJets_Gpt_1j_5f_NLO/GJets_1j_Gpt-50To100_5f_NLO/GJets_1j_Gpt-50To100_5f_NLO_run_card.dat (hunk 2 above)
Comment (idebruyn) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/GJets_Gpt_1j_5f_NLO/GJets_1j_Gpt-50To100_5f_NLO/GJets_1j_Gpt-50To100_5f_NLO_run_card.dat]: Thanks for spotting the mistake! Fixed now.

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/GJets_Gpt_1j_5f_NLO/GJets_1j_Gpt-50To100_5f_NLO/GJets_1j_Gpt-50To100_5f_NLO_run_card.dat (hunk 1 above)
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/GJets_Gpt_1j_5f_NLO/GJets_1j_Gpt-50To100_5f_NLO/GJets_1j_Gpt-50To100_5f_NLO_run_card.dat]: can you please answer also this before to merge?

**Comment by idebruyn** [comment]
The Gjets run card should be correct now. Or am I missing something?

**Comment by perrozzi** [comment]
yes it looks so, I had missed the update. merging

**Pull request merged**